src/utils/testing/mocks/mockJourneyInstance.ts

```diff
@@ -6,11 +6,18 @@ const journeyInstance: ZetkinJourneyInstance = {
   assigned_to: [person],
   created_at: '2022-04-01T03:29:12+02:00',
   id: 333,
+  journey: {
+    id: 1,
+    title: 'Training',
+  },
+  milestones: [],
   next_milestone: {
     deadline: '2022-04-18T00:29:12+02:00',
+    status: '',
     title: 'perform lip sync',
   },
```

Comment:
I think if there's a `next_milestone` it would be good to have that item in `milestones`. Maybe inside of this file you can make a mock generator for milestones and use it in both places :)

Reply:
Even though this is a temporary milestone object, this is probably useful to have separate so we can create milestones easily and then you can just update wherever `mockMilestone` is used instead of every object.